app/src/main/java/ca/bc/gov/bchealth/ui/setting/SettingsFragment.kt

```diff
@@ -137,7 +137,7 @@ class SettingsFragment : BaseFragment(R.layout.fragment_settings) {
                         return@collect
                     } else {
                         initClickListeners()
-                        val isLoginStatusActive = it.loginStatus == LoginStatus.ACTIVE
+                        val isLoginStatusActive = (it.loginStatus != null && it.loginStatus == LoginStatus.ACTIVE)
```

Comment:
If a LoginStatus is ACTIVE, it's already != null,  isn't it?

Reply:
@bsavoini-fw  Initial state of LoginStatus is declared as  val loginStatus: LoginStatus? = null,
this means when your fragment start listening it you will receive it null for the first time.
that was impacting settings fragment.